### DISCUSSION TITLE
Is there a way to control laptop screen brightness?

### DISCUSSION THREAD

**Discussion by EthanHarp** [manna-harbour/miryoku#357] [Q&A] [answered]
Hello, new miryoku/chocofi user here. I was wondering if there is a way to control laptop screen brightness that I was missing in the reference manual? If not then I was wondering what the easiest way would be for someone who has never used qmk other than for flashing pre-existing firmware? Thanks

Edit: If I would need to add it manually is there a recommended place for it? My keyboard doesn't have rgb or bluetooth so something on the media layer was my initial thought

**Comment by manna-harbour** [comment, marked as answer]
The host operating system may not have support for setting laptop brightness from an external keyboard. If you can find a key or key combination that works, you can add them to unused keys with https://github.com/manna-harbour/miryoku/discussions/85, or if you can find a utility to do it you can bind the utility to a keyboard shortcut in your OS.

What is the OS? In Windows you can use Win+A to bring up a keyboard controllable brightness slider.

Also see https://community.keyboard.io/t/what-are-the-codes-for-the-brightness-control-keys/4094 and https://docs.qmk.fm/keycodes_basic#media-keys.

**Reply by EthanHarp** [reply]
The host OS is Linux and my current brightness utility listens for XF86MonBrightnessUp and XF86MonBrightnessDown, based on the link you sent would KC_BRIGHTNESS_UP and KC_BRIGHTNESS_DOWN be their equivalents?

**Reply by EthanHarp** [reply]
Just to follow up I had the time to try out  KC_BRIGHTNESS_UP and KC_BRIGHTNESS_DOWN following the instructions for substituting a layer and it works as expected.
